sopel/tools/__init__.py

```diff
@@ -197,18 +235,14 @@ def new(*args, **kwargs):
     new.__name__ = old.__name__
     return new
 
-
 # from
 # http://parand.com/say/index.php/2007/07/13/simple-multi-dimensional-dictionaries-in-python/
-# A simple class to make mutli dimensional dict easy to use
+# A simple class to make multi-dimensional dict easy to use
```

Comment:
If it's an above-the-class in-line comment, it can be a docstring for the `Ddict` class below.